gossiper/gossiper_test.go

```diff
@@ -0,0 +1,16 @@
+package gossiper
+
+import "testing"
+
+func TestAddRemovePeer(t *testing.T)  {
+	g := new(gossiper)
+	g.name = "test"
+	g.AddPeer(gossiper{"peer", "localhost", make(map[string]gossiper)})
+	if _, ok := g.peers["peer"]; !ok {
+		t.Errorf("Add Failed")
```

Comment:
Might wanna look at testing package -> https://godoc.org/github.com/stretchr/testify/assert and https://godoc.org/github.com/stretchr/testify/require

Reply:
will do